themes/socialbase/templates/group/group--hero.html.twig

```diff
@@ -28,7 +28,19 @@
   ]
 %}
 
-<div{{ attributes.addClass(cover_classes) }} {% if group_hero_styled_image_url %} style="background-image: url('{{ group_hero_styled_image_url }}');" {% endif %}>
+{%
+  set cover_wrap_classes = group_hero_small ? 'cover-wrap small' : 'cover-wrap'
+%}
+
+{%
+  set hero_image = group_hero_styled_image_url
```

Comment:
@jochemvn Why not just set correct image style to `group_hero_styled_image_url` in preprocess?

Reply:
yeah... that sounds like a pretty good idea 😅... Fix incoming

Reply:
@ribel I've made that change... Can you quickly check? 😇

Reply:
Yes, looks much better now 👍